staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -487,7 +494,7 @@ func (w *watchCache) GetAllEventsSinceThreadUnsafe(resourceVersion uint64) ([]*w
 		rev := w.revCache[(w.startIndex+first+i)%w.capacity]
 		result, isOK := w.cache[rev]
 		if !isOK {
-			klog.Errorf("Expected event not found in cache. Rev %v", rev)
+			return nil,fmt.Errorf("Expected event not found in cache. Rev %v", rev)
```

Comment:
I think go fmt would complain

Reply:
I already ran "gofmt" on this changed file.  This is the file after formatting.

Reply:
You are right. I ran gofmt but didn't add "-w" parameter so the change wasn't applied.  Will fix.  Thanks for catching this.